src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -115,6 +115,9 @@
     "good": PREFIX + 233,
     "bad": PREFIX + 234,
     "meh": PREFIX + 235,
+    "abnormal_mechanism": PREFIX + 236,  # release health
+    "anr_foreground": PREFIX + 237,  # release health
+    "anr_background": PREFIX + 238,  # release health
```

Comment:
Have you checked that these strings have not yet popped up in either `sentry_stringindexer` or `sentry_perfstringindexer` tables? We might get unexpected behavior when there is both a static string and a db entry for an org.

https://github.com/getsentry/sentry/blob/4a61c0e5bf9662045be0379fcc799f0fa0e58fa7/src/sentry/sentry_metrics/indexer/strings.py#L18-L19

Reply:
These values aren't in the `sentry_perfstringindexer` or `sentry_stringindexer` tables!

```
select string, count(*) from <table>
where string IN ('anr_foreground', 'anr_background', 'abnormal_mechanism')
group by string
```